webapp/channels/src/components/admin_console/system_users/system_users_list_actions/index.tsx

```diff
@@ -89,7 +116,43 @@ function AdminConsoleListActions(props: Props) {
                         defaultMessage='Activate'
                     />
                 }
-            />
+                onClick={async () => {
+                    // TODO: disable if this is true
+                    if (user.auth_service === Constants.LDAP_SERVICE) {
+                        return;
+                    }
+
+                    const {error} = await dispatch(updateUserActive(user.id, true));
+                    if (error) {
+                        onError(error);
+                    }
+
+                    // TODO: add callback to update the user's active status
```

Comment:
I have a whole bunch of TODOs here for having the user object that the list uses be updated with new information. Right now since we hold all of the user report lines in state, we would have to change a callback all the way up to update that state.

But I'm starting to think that maybe we might want to store these results in Redux somewhere? One reason being that `UserReport` now fully implements `UserProfile`, so if we wanted we could have the reporting interface use the objects from there, and we could update those. Then we store the more specific report data somewhere else.